tools/ci-ts/tests/flux-monitor.test.ts

```diff
@@ -1,96 +1,156 @@
+let linkToken: contract.Instance<contract.LinkTokenFactory>
+let fluxAggregator: contract.Instance<FluxAggregatorFactory>
+let node1Address: string
+let node2Address: string
+
+async function assertAggregatorValues(
+  latestAnswer: number,
+  latestRound: number,
+  reportingRound: number,
+  latestSubmission1: number,
+  latestSubmission2: number,
+  msg: string,
+): Promise<void> {
+  const [la, lr, rr, ls1, ls2] = await Promise.all([
+    fluxAggregator.latestAnswer(),
+    fluxAggregator.latestRound(),
+    fluxAggregator.reportingRound(),
+    fluxAggregator.latestSubmission(node1Address).then(res => res[1]),
+    fluxAggregator.latestSubmission(node2Address).then(res => res[1]),
+  ])
+  matchers.bigNum(latestAnswer, la, `${msg} : latest answer`)
+  matchers.bigNum(latestRound, lr, `${msg} : latest round`)
+  matchers.bigNum(reportingRound, rr, `${msg} : reporting round`)
+  matchers.bigNum(latestSubmission1, ls1, `${msg} : node 1 latest submission`)
+  matchers.bigNum(latestSubmission2, ls2, `${msg} : node 2 latest submission`)
 }
 
-async function assertJobRun(
-  jobId: string,
-  count: number,
-  errorMessage: string,
-) {
-  await assertAsync(() => {
-    const jobRuns = clClient.getJobRuns()
-    const jobRun = jobRuns[jobRuns.length - 1]
-    return (
-      clClient.getJobRuns().length === count &&
-      jobRun.status === 'completed' &&
-      jobRun.jobId === jobId
-    )
-  }, errorMessage)
+async function assertLatestAnswerEq(n: number) {
+  await assertAsync(
+    async () => (await fluxAggregator.latestAnswer()).eq(n),
+    `latestAnswer should eventually equal ${n}`,
+  )
 }
 
-describe('flux monitor eth client integration', () => {
-  let linkToken: contract.Instance<contract.LinkTokenFactory>
-  let fluxAggregator: contract.Instance<FluxAggregatorFactory>
-  let job: JobSpec
-  let node1Address: string
-
-  beforeAll(async () => {
-    clClient.login()
-    node1Address = clClient.getAdminInfo()[0].address
-    await fundAddress(carol.address, 5)
-    await fundAddress(node1Address, 5)
-    linkToken = await linkTokenFactory.deploy()
-    await linkToken.deployed()
-  })
+beforeAll(async () => {
+  printHeading('Setup')
+  clClient1.login()
+  clClient2.login()
+  node1Address = clClient1.getAdminInfo()[0].address
+  node2Address = clClient2.getAdminInfo()[0].address
+  await fundAddress(carol.address)
+  await fundAddress(node1Address)
+  await fundAddress(node2Address)
+  linkToken = await linkTokenFactory.deploy()
+  await linkToken.deployed()
+  console.log(`Chainlink Node 1 address: ${node1Address}`)
+  console.log(`Chainlink Node 2 address: ${node2Address}`)
+  console.log(`Contract creator's address: ${carol.address}`)
+  console.log(`Deployed LinkToken contract: ${linkToken.address}`)
+})
 
-  afterEach(async () => {
-    clClient.archiveJob(job.id)
-    await changePriceFeed(100) // original price
-  })
+beforeEach(async () => {
+  printHeading('Running Test')
+  fluxMonitorJob = cloneDeep(_fluxMonitorJob)
```

Comment:
This is just JSON right? If so, we can drop the lodash dependency and use `JSON.parse(JSON.stringify(_fluxMonitorJob))`

Reply:
okay! nice little hack